### ISSUE TITLE
Gene expression data missing

### ISSUE THREAD

**Issue by lynnpais** [broadinstitute/seqr#2942]
**Describe the bug**
I am no longer seeing gene expression data in cases with RNASeq data loaded. 

**Link to page(s) where bug is occurring**
https://seqr.broadinstitute.org/project/R0405_cmg_vcgs_genomes/saved_variants/variant/SV0107086_1235196334_f024852_v

**Scope of the bug**
seqr-wide I think 

**Screenshots**
Old image from a presentation - 
<img width="882" alt="Screen Shot 2022-09-07 at 2 49 27 PM" src="https://user-images.githubusercontent.com/37872186/188955148-6fd4943e-e914-4e22-b494-eb2c02f37720.png">

Current view - 
<img width="1244" alt="Screen Shot 2022-09-07 at 2 50 18 PM" src="https://user-images.githubusercontent.com/37872186/188955217-31b9d250-b676-415e-8d41-a0e154cfce66.png">

**Comment by hanars** [comment]
The show RNA outlier button is also missing for this family on the family page, so seems to be an issue with all RNA data not just expression data

**Comment by lynnpais** [comment]
I saw the links on the Family page for RGP_248 so wasn't sure 
<img width="1278" alt="Screen Shot 2022-09-07 at 3 03 29 PM" src="https://user-images.githubusercontent.com/37872186/188957317-0f7f7b8e-fc06-4b5e-b797-7ca39a54be0e.png">

**Comment by hanars** [comment]
There was a bug in out RNA loading process in which we deleted previously loaded data during loading even if it wasn't present in the latest file. We fixed this bug back in May, but it looks like I missed the fact that while this bug was live we did delete some data, it appears about 100 samples were affected. Luckily I still have the old data Ben sent me so I was able to add this data back, so that should be resolved now

Unrelatedly, there is also a bug for showing the "show gene expression" button on SVs. So if you look at the variants in this family  now you will see the outrider data and the "show expression" button on all the SNPs, but the button is still missing on the Sv you linked on this ticket. That is a pretty straightforward bug fix, and will be resolved I would say by Monday, Tuesday at the latest
